### PR TITLE
DOC Add validations related to DBField

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-framework/issues/11403

Linting issue is existing

### CODE REVIEW

File: en/08_Changelogs/5.4.0.md [hunk 1]
```diff
@@ -102,6 +102,8 @@ SilverStripe\ORM\FieldType\DBPolymorphicForeignKey:
 - [`HTTPOutputHandler::isCli()`](api:SilverStripe\Logging\HTTPOutputHandler::isCli()) has been deprecated. Use [`Director::is_cli()`](api:SilverStripe\Control\Director::is_cli()) instead instead.
 - [`PasswordValidator`](api:SilverStripe\Security\PasswordValidator) has been deprecated. It will be renamed to `SilverStripe\Security\Validation\RulesPasswordValidator`.
 - [`ContentReviewEmails::isValidEmail()`](api:SilverStripe\ContentReview\Tasks\ContentReviewEmails::isValidEmail()) has been deprecated. Use [`Email::is_valid_address()`](api:SilverStripe\Control\Email\Email::is_valid_address()) instead.
+- [`DBField.defaultVal`](api:SilverStripe\ORM\FieldType\DBField->defaultVal). Use getDefaultValue() and setDefaultValue() instead.
```
Comment (ChloeSartorelli) [inline en/08_Changelogs/5.4.0.md]:
```suggestion
- The [`DBField.defaultVal`](api:SilverStripe\ORM\FieldType\DBField->defaultVal) property has been deprecated. Use getDefaultValue() and setDefaultValue() instead.
```
Also make `getDefaultValue()` and `setDefaultValue()` into links.

File: en/08_Changelogs/5.4.0.md [hunk 2]
```diff
@@ -102,6 +102,8 @@ SilverStripe\ORM\FieldType\DBPolymorphicForeignKey:
...
+- [`DBFile::validate()`](api:SilverStripe\Assets\Storage\DBFile::validate()) has been deprecated. Use validateFilename() instead.
```
Comment (ChloeSartorelli) [inline en/08_Changelogs/5.4.0.md]: `validateFilename()` should be a link.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: en/08_Changelogs/5.4.0.md (hunk 1 above)
Comment (emteknetnz) [reply en/08_Changelogs/5.4.0.md]: Updated

File: en/08_Changelogs/5.4.0.md (hunk 2 above)
Comment (emteknetnz) [reply en/08_Changelogs/5.4.0.md]: Updated

**Review by GuySartorelli — APPROVED** [review]
LGTM

**Pull request merged**